lib/src/visitor/perform.dart

```diff
@@ -555,11 +566,35 @@ class _PerformVisitor
     });
   }
 
+  /// If the scheme is `package`, converts the [import] url into a dart package uri.
+  /// If the scheme is not `package`, returns the the same [import] url.
+  /// If the [_packageResolver] is null, throws a [SassRuntimeException].
+  /// If the [_packageResolver] cannot find the file, throws a [SassRuntimeException].
+  Uri _resolvePackageUri(DynamicImport import) {
+    var packageUri = import.url;
+    if (packageUri.scheme == 'package') {
+      if (_packageResolver == null) {
+        throw _exception(
+            'Can\'t resolve: "$packageUri", packageResolver is not supported by node vm.'
+            ' If you are using dart-vm please provide a `SyncPackageResolver` to the `render` function',
+            import.span);
+      }
+
+      var resolvedPackageUri = _packageResolver.resolveUri(packageUri);
+      if (resolvedPackageUri == null ||
+          !dirExists(p.dirname(p.fromUri(resolvedPackageUri)))) {
```

Comment:
This check is for when the package URI is not resolvable. What should I do instead this check?

If I remove this check `resolvedPackageUri` will be null, which in turn will throw:
```
NoSuchMethodError: The getter 'scheme' was called on null.
```

Second option would be return an empty uri, which was what I have at the beginning and you told me to better add a check and throw a better error. Please see the comment you added on https://github.com/sass/dart-sass/pull/53#discussion-diff-92956954R524

Third option would be return the original `packageUri`, but that lets to previous problem on having an error message not showing good information.

Finally the line 585 in https://github.com/sass/dart-sass/pull/53#discussion-diff-96078273R585 was added because previous version of sdk returned an URI even when there was no existing file.